charts/newrelic-pixie/README.md

```diff
@@ -0,0 +1,49 @@
+# newrelic-pixie
+
+## Chart Details
+
+This chart will deploy the New Relic Pixie Integration.
+
+IMPORTANT: this chart has to be deployed in the same namespace as Pixie. This is required because
+it needs to access the cluster id inside the Pixie secrets. By default, Pixie is installed in the `pl` namespace.
+
+## Configuration
+
+| Parameter                     | Description                                                  | Default                    |
+| ----------------------------- | ------------------------------------------------------------ | -------------------------- |
+| `global.cluster` - `cluster`  | The cluster name for the Kubernetes cluster. Required.       |                            |
+| `global.licenseKey` - `licenseKey` | The New Relic license key (stored in a secret). Required.    |                            |
+| `apiKey`                      | The Pixie API key (stored in a secret). Required.            |                            |
+| `verbose`                     | Whether the integration should run in verbose mode or not.   | false                      |
+| `global.customSecretName` - `customSecretName` | Name of the Secret object where the New Relic license and Pixie API key are stored                                                                                                                                                                         |                                 |
```

Comment:
From the readme I did not get clearly that these were _alternatives_ to provide the values directly. Perhaps we can clarify it a bit?

Reply:
I'll add a section for this 👍